contracts/proxy/transparent/TransparentUpgradeableProxy.sol

```diff
@@ -70,37 +68,30 @@ contract TransparentUpgradeableProxy is ERC1967Proxy {
     error ProxyDeniedAdminAccess();
 
     /**
-     * @dev msg.value is not 0.
+     * @dev The proxy rejected value included with `upgradeToAndCall` that would not execute the `receive` function.
      */
-    error ProxyNonPayableFunction();
+    error ProxyRejectedValue();
```

Comment:
Is this needed? I don't see where it's used

Reply:
Left from an earlier version. Removed now.